sql/schema_changer.go

```diff
@@ -196,10 +196,7 @@ func isSchemaChangeRetryError(err error) bool {
 
 // Execute the entire schema change in steps. startBackfillNotification is
 // called before the backfill starts; it can be nil.
-func (sc SchemaChanger) exec(
-	startBackfillNotification func() error,
-	oldNameNotInUseNotification func(),
-) error {
+func (sc SchemaChanger) exec(testingKnobs *ExecutorTestingKnobs) error {
```

Comment:
Can we make the testing knobs an attribute of the `SchemaChanger` instead of passing them through the stack? That's what we do with testing knobs in most other places.